docs/config.md

```
@@ -362,6 +373,10 @@ Service configuration.
 
 TLS configuration.
 
+See also:
+- https://fastapi.tiangolo.com/deployment/https/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)